### PR TITLE
Collapse static handler tests

### PR DESCRIPTION
**Pull request by danilvpetrov**
#### What change does this introduce?

This PR collapses all static analysis handler tests into a single folder  `static/testdata/handlers`.

#### Why make this change?

This change is required to group the handler tests and reduce the number of test data and duration of the tests.

#### Is there anything you are unsure about?

No

#### What issues does this relate to?

Relates to https://github.com/dogmatiq/configkit/issues/104.

### CODE REVIEW

File: static/handler_test.go [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+	identities ...configkit.Identity,
+) {
+	for _, identity := range identities {
+		_, ok := hs.ByIdentity(identity)
+		Expect(ok).To(BeTrue())
+	}
+}
+
+var _ = Describe("func FromPackages() (handler analysis)", func() {
+	When("the application contains a single handler handler of a kind", func() {
```
Comment (jmalloc) [inline static/handler_test.go:26]:
```suggestion
	When("the application contains a single handler of each type", func() {
```

File: static/handler_test.go [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+		It("returns the a single configuration for each handle kind", func() {
```
Comment (jmalloc) [inline static/handler_test.go:27]:
```suggestion
		It("returns a single configuration for each handler type", func() {
```

File: static/handler_test.go [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+						cfixtures.MessageCTypeName: message.EventRole,
+						cfixtures.MessageDTypeName: message.EventRole,
+					},
+				},
+			))
+		})
+	})
+
+	When("the application contains multiple aggregate handlers", func() {
+		It("returns all of the aggregate handler configurations", func() {
```
Comment (jmalloc) [inline static/handler_test.go:163]:
```suggestion
	When("the application multiple handlers of each type", func() {
		It("returns all of the handler configurations", func() {
```

File: static/handler_test.go [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+				},
+				configkit.Identity{
+					Name: "<second-integration>",
+					Key:  "6bed3fbc-30e2-44c7-9a5b-e440ffe370d9",
+				},
+			)
+		})
+	})
+
+	When("a nil value is passed as an aggregate handler", func() {
```
Comment (jmalloc) [inline static/handler_test.go:213]:
```suggestion
	When("a nil value is passed as a handler", func() {
```

File: static/handler_test.go [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+		It("does not add an aggregate handler to the application configuration", func() {
```
Comment (jmalloc) [inline static/handler_test.go:214]:
```suggestion
		It("does not add a handler to the application configuration", func() {
```

File: static/handler_test.go [hunk 6]
```diff
@@ -0,0 +1,348 @@
...
+			Expect(err).NotTo(HaveOccurred())
+
+			apps := FromPackages(pkgs)
+			Expect(apps).To(HaveLen(1))
+			Expect(apps[0].Handlers()).To(Equal(configkit.HandlerSet{}))
+		})
+	})
+
+	When("a nil value is passed as a message", func() {
+		It("does not add the message to the aggregate configuration", func() {
```
Comment (jmalloc) [inline static/handler_test.go:230]:
```suggestion
		It("does not add the message to the handler configuration", func() {
```

**Review by jmalloc — CHANGES_REQUESTED** [review]
Just some test descriptions need to be fixed up... otherwise great, much smaller!

**Comment by danilvpetrov** [comment]
Ok, that's done, please let me know if there is anything else left to fix.

**Comment by jmalloc** [comment]
Nah man, that's it.. merge away!

**Pull request merged**